js/src/sdk/base.toolset.ts

```diff
@@ -29,6 +25,14 @@ export class ComposioToolSet {
     localActions: IPythonActionDetails["data"] | undefined;
     customActionRegistry: ActionRegistry;
 
+    processors: {
```

Comment:
Consider adding JSDoc comments for the processor functions to better document their purpose and expected behavior:

```typescript
/**
 * Pre-processor function to transform input before execution
 * @param {Record<string, any>} input - The input data to transform
 * @returns {Record<string, any>} The transformed input
 */
pre: (input: Record<string, any>) => input,

/**
 * Post-processor function to transform output after execution
 * @param {Record<string, any>} input - The output data to transform
 * @returns {Record<string, any>} The transformed output
 */
post: (input: Record<string, any>) => input
```